python/plugins/db_manager/dlg_table_properties.py

```diff
@@ -333,25 +333,37 @@ def deleteIndex(self):
                 DlgDbError.showError(e, self)
 
     def createComment(self):
-        #Function that add a comment to the selected table
+        """Adds a comment to the selected table"""
+
         try:
             #Using the db connector, executing de SQL query Comment on table
             self.db.connector._execute(None, 'COMMENT ON TABLE "{0}"."{1}" IS E\'{2}\';'.format(self.table.schema().name, self.table.name, self.viewComment.text()))
         except DbError as e:
             DlgDbError.showError(e, self)
             return
+        except Exception as e:
```

Comment:
@elpaso is there any chance to avoid the catch all exception? I think even for quick and dirty patches this should be avoided.